src/lib/storage/proposal/settings.rb

```diff
@@ -82,6 +84,7 @@ def initialize
           # Not yet in control.xml
           @home_min_size                 = DiskSize.GiB(10)
           @home_max_size                 = DiskSize.unlimited
+          @useful_free_space_min_size    = DiskSize.MiB(30)
```

Comment:
Not sure if we want to move this responsibility to control.xml that is typically written by a PM.

Things might easily break internally if we leave this open for outside configuration to somebody who doesn't know the implications of this.